ops/Dockerfile.server

```
@@ -0,0 +1,35 @@
+COPY . .
+RUN npm run --workspace server build || echo "Skipping server build (JS runtime)"
+
+# Runtime image only carries the compiled output and production deps.
+FROM base AS runner
+WORKDIR /app
+
+ENV NODE_ENV=production
+ENV PORT=4000
+
+COPY --from=builder /app/server/package.json ./package.json
+COPY --from=builder /app/server/dist ./dist 2>/dev/null || true
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)